lib/generators/my_zipcode_gem/templates/county_model.rb

```diff
@@ -8,8 +8,17 @@ class County < ActiveRecord::Base
   validates :name, uniqueness: { scope: :state_id, case_sensitive: false },
                    presence: true
 
-  scope :without_zipcodes, -> { joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL") }
-  scope :without_state, -> { where("state_id IS NULL") }
+  scope :without_zipcodes, lambda {
+    county = County.arel_table
+    zipcodes = Zipcode.arel_table
+    zipjoin = county
+        .join(zipcodes, Arel::Nodes::OuterJoin)
+        .on(zipcodes[:county_id].eq(county[:id]))
```

Comment:
Looks like `Style/DotPosition` is the rule. It looks like leading is the default in rubocop though.

Maybe Hound overrides it and it needs to be manually set.

```
# Multi-line method chaining should be done with leading dots.
Style/DotPosition:
  EnforcedStyle: leading
```